### DISCUSSION TITLE
How does a Live Table work?

### DISCUSSION THREAD

**Discussion by BlackBearFTW** [spectreconsole/spectre.console#679] [Q&A] [answered]
I tried doing AnsiConsole.Live, but .Live is not being recognized as a method.

**Comment by ryandle** [comment, marked as answer]
@BlackBearFTW check to make sure you have the latest version of the Spectre.Console nuget package. It is definitely there:

https://github.com/spectreconsole/spectre.console/blob/main/src/Spectre.Console/AnsiConsole.Live.cs

Here's an example:

https://github.com/spectreconsole/spectre.console/blob/main/examples/Console/LiveTable/Program.cs

**Reply by BlackBearFTW** [reply]
Thank you for your answer, sorry for replying so late. 

I was also wondering if its possible to modify an existing row, to display a different value?

**Reply by ryandle** [reply]
Yes, I think that would work by updating the row data in thew table and then calling `ctx.Refresh();` like in the above example.
